performance_manager/lib/rt_vehicle_positions.py

```diff
@@ -0,0 +1,277 @@
+import logging
+
+import pandas
+import numpy
+import sqlalchemy
+import re
+import datetime
+
+from sqlalchemy import select
+from sqlalchemy import update
+from sqlalchemy import insert
+from sqlalchemy import delete
```

Comment:
i started using `import sqalchemy as sa` and prefixing everything i used.

Reply:
Incorporated